### PR TITLE
Update for elementary OS 6 Odin

### LINKED ISSUE #4: Distribute as Flatpak for AppCenter
https://github.com/flathub/flathub/wiki/App-Submission

### PR DESCRIPTION
**Pull request by meisenzahl**
Would be great if Ideogram would also work on OS 6 😍️. Fixes #4.

Should be enough for it to

- work as a Flatpak 📦️
- support dark style 🌑️

### CODE REVIEW

**Comment by cassidyjames** [comment]
There's more than just Flatpaking required, and I'm not 100% sure this app will work sandboxed:

- [ ] Keyboard shortcut not set up
- [ ] Copying to clipboard doesn't work
- [ ] Auto pasting doesn't work

**Comment by cassidyjames** [comment]
@meisenzahl it seems everything but the keyboard shortcut works. Do we know if it's possible to configure keyboard shortcuts, and if not, I suppose I could just have a note in the app with a link to the settings.

**Comment by meisenzahl** [comment]
@cassidyjames according to https://github.com/alainm23/planner/issues/246 it _could_ be possible to configure keyboard shortcuts. The last commits I pushed were based on the changes in Planner. I need to investigate this further.

**Comment by cassidyjames** [comment]
@meisenzahl considering that's pretty hacky (and might not pass AppCenter review, to be honest), I think I'll lean on informing the user how to add the command themselves with a link to the settings.

**Comment by leggettc18** [comment]
I may be missing something, but I pulled the forked repo's Odin branch and build the flatpak and the auto-paste isn't working for me. Am I the only one?

**Comment by meisenzahl** [comment]
> @meisenzahl considering that's pretty hacky (and might not pass AppCenter review, to be honest), I think I'll lean on informing the user how to add the command themselves with a link to the settings.

@cassidyjames do you want to implement this in a follow-up PR?

Let me know what else I can do to get Ideogram into AppCenter 😃

File: com.github.cassidyjames.ideogram.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+runtime-version: '6'
+sdk: io.elementary.Sdk
+command: com.github.cassidyjames.ideogram
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+  # needed for perfers-color-scheme
+  - '--system-talk-name=org.freedesktop.Accounts'
```
Comment (cassidyjames) [inline com.github.cassidyjames.ideogram.yml:12]: This isn't needed with newer Granite, now

```suggestion
```

**Comment by cassidyjames** [comment]
@meisenzahl I do think I'd like to solve the keyboard shortcut prompt before releasing for OS 6 as otherwise it will feel pretty broken. But I merged this work in; thank you!

**Comment by subnut** [comment]
@cassidyjames Ideogram doesn't seem to be available on elementary OS 6.1 right now. How to install it?

**Pull request merged**